adal/src/main/java/com/microsoft/aad/adal/AuthenticationResult.java

```diff
@@ -305,6 +307,10 @@ public boolean isExpired() {
         return TokenCacheItem.isTokenExpired(getExpiresOn());
     }
 
+    public final String getAuthority() {
```

Comment:
your getAuthority returns the value set in setcloudInstanceName. this is quite confusing. As per the comment above, this may be incorrect nomenclature too. you should be returning the complete authority URL.